tests/fmt.rs

```diff
@@ -11,6 +11,28 @@ test! {
   status: EXIT_FAILURE,
 }
 
+test! {
+  name: dry_run_ok,
+  justfile: "",
+  args: ("--unstable", "--fmt", "--dry-run"),
```

Comment:
I think this should be a more complicated justfile that has no diff, so at least one assignment and one recipe, with some text in the body.

Reply:
True, done